WordPress/src/main/java/org/wordpress/android/ui/compose/theme/AppTheme.kt

```diff
@@ -0,0 +1,23 @@
+package org.wordpress.android.ui.compose.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.MaterialTheme
+import androidx.compose.runtime.Composable
+import org.wordpress.android.BuildConfig
+import org.wordpress.android.ui.compose.theme.color.JetpackColors
+import org.wordpress.android.ui.compose.theme.color.WordPressColors
+
+/**
+ * Project's base theme.
+ */
+@Composable
+fun AppTheme(
+    isDarkTheme: Boolean = isSystemInDarkTheme(),
+    content: @Composable () -> Unit
+) {
+    val themeColors = if (BuildConfig.IS_JETPACK_APP) JetpackColors() else WordPressColors()
+    MaterialTheme(
+            colors = if (isDarkTheme) themeColors.dark() else themeColors.light(),
+            content = content
+    )
+}
```

Comment:
I'm not sure if you'll need a similar thing or not, but just sharing how we did in WCAndroid:

In Compose, some components [support](https://developer.android.com/jetpack/compose/themes/material#content-color) the usage of `contentColor`, and some not, when the Layout supports it, it makes it easier to have the correct color in different themes (light vs dark), instead of having to set the colors manually for each child Composable.

For this case, the theme we use in WCAndroid [adds](https://github.com/woocommerce/woocommerce-android/blob/d5c0e6f9d7acbbd2fcabcf8845335a79aab6e745/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/compose/theme/Theme.kt#L16) by default a `Surface` with the correct background theme, and this way, the different Composables in each screen define the correct `contentColor` that matches the theme.

Reply:
We're probably not needing it for this PR but I'll add it to our private slack channel to keep this in mind since it's a great find that should become helpful in the near future of our Compose journey 🚀 .

Thank you!

Reply:
We actually need this too (see https://github.com/wordpress-mobile/WordPress-Android/issues/17085), thank you @hichamboushaba for the feedback 🙇